src/util.py

```diff
@@ -260,12 +265,110 @@ def xr_add_cyclic_longitudes(da, coord):
 
     return new_da
 
+
+def findLocalExtrema(da, highVal=0, lowVal=1000, eType='Low'):
+    """
+    Utility function to find local low/high field variable coordinates on a contour map. To classify as a local high, the data
+    point must be greater than highVal, and to classify as a local low, the data point must be less than lowVal.
+
+    Args:
+        da: (:class:`xarray.DataArray`):
+            Xarray data array containing the lat, lon, and field variable (ex. pressure) data values
+        highVal (:class:`int`):
+            Data value that the local high must be greater than to qualify as a "local high" location.
+            Default highVal is 0.
+        lowVal (:class:`int`):
+            Data value that the local low must be less than to qualify as a "local low" location.
+            Default lowVal is 1000.
+        eType (:class:`str`):
+            'Low' or 'High'
+            Determines which extrema are being found- minimum or maximum, respectively.
+            Default eType is 'Low'.
+    Returns:
+        clusterExtremas (:class:`list`):
+            List of coordinate tuples in GPS form (lon in degrees, lat in degrees)
+            that specify local low/high locations
+    """
+    
+    import numpy as np
+    from sklearn.cluster import DBSCAN
```

Comment:
You should add `scikit-learn` dependency into **setup.py** in the root directory.

Reply:
Done!

Reply:
Looks good! As soon as @clyne approves, we will merge this PR.